shipLHC/run_simSND.py

```diff
@@ -162,9 +170,27 @@ def Exec(self,opt):
   myPgun = ROOT.FairBoxGenerator(options.pID,1)
   myPgun.SetPRange(options.Estart,options.Eend)
   myPgun.SetPhiRange(0, 360) # // Azimuth angle range [degree]
-  myPgun.SetXYZ(options.EVx*u.cm, options.EVy*u.cm, options.EVz*u.cm) 
   myPgun.SetThetaRange(0,0) # // Polar angle in lab system range [degree]
+  if options.multiplePGSources:
+    # multiple PG sources in the x-y plane; z is always the same!
+    myPgun.SetBoxXYZ(options.EVx*u.cm,
+                     options.EVy*u.cm,
+                     (options.EVx+options.Dx)*u.cm,
+                     (options.EVy+options.Dy)*u.cm,
+                     options.EVz*u.cm)
+  else:
+     # point source
+     myPgun.SetXYZ(options.EVx*u.cm, options.EVy*u.cm, options.EVz*u.cm)
   primGen.AddGenerator(myPgun)
+  # To generate particle guns along the z axis, create z *target* layers with a set step
+  # For an **unknown** reason simply setting target z thickness doesn't produce the expected result
+  if options.multiplePGSources:
+    targetZpos = np.array(np.arange(options.nZSlices)*options.zSliceStep*u.cm, dtype='d')
+    primGen.SetMultTarget(len(targetZpos), targetZpos, 0*u.cm) # dummy thickness set to 0
+  print(f'===> Setting particle gun sources starting at (x1,y1,z1)=('+
+        f'{options.EVx},{options.EVy},{options.EVz})[cm × cm × cm] \n'+
+        f'with a uniform x-y spread of (Dx,Dy)=({options.Dx},{options.Dy})[cm × cm]'+
+        f' and {options.nZSlices} z slices in steps of {options.zSliceStep}[cm].')
```

Comment:
```suggestion
  print('===> Setting particle gun sources starting at (x1,y1,z1)='
        f'({options.EVx},{options.EVy},{options.EVz})[cm × cm × cm] \n'
        f'with a uniform x-y spread of (Dx,Dy)=({options.Dx},{options.Dy})[cm × cm]'
        f' and {options.nZSlices} z slices in steps of {options.zSliceStep}[cm].')
```
The +s are not needed, as we are within parentheses, so the string is just implicitly concatenated.

Would spacing around the commas and equals mess up the formatting? (In general, we could probably use automatic wrapping for logging, so that we don't need to worry about this, but no need to figure out how to do that now).